content/departments/engineering/dev/process/releases/procedure.md

```diff
@@ -0,0 +1,69 @@
+# Major / minor release procedures
+
+## Code freeze
+
+The goal of code freeze is to stabilize the release for the final build, and to mitigate risk with each change by fixing bugs caught during the QA process, or making improvements that increase stability and quality in the final build.
+
+During the code freeze we will typically cut a new candidate each day and deploy it to the test instance for additional QA. If additional candidates are necessary, reach out to the release guild.
+
+There are 3 distinct sections of code freeze:
+
+1. open code freeze
+2. closed code freeze
+3. final build freeze
+
+### Open code freeze
+
+During open code freeze, the release branch is open to changes and will require changes to be backported from `main` using our backport tooling. It is up to the individuals discretion to determine whether a change should be included in the final build. Release captains may monitor the backported changes for anything that seems risky and may reach out to understand more about the requirements. It is not a given that every change can be backported during this period. Substantial changes should be completed and merged **before** the code freeze period has started.
+
+### Closed code freeze
+
+A few days before the release the branch will be closed to changes and will require a member of the release guild to merge backports. The goal of this step is to stabalize a final build. During this period release captains will exercise discretion whether a change warrants backport. Some guidelines for things that are eligible:
+
+1. Documentation
+2. Bug fixes
+3. Release infrastructure (lint, generate fixup, etc)
+4. Low risk changes (copy, colors, etc)
```

Comment:
should we not have the same criteria listed out for the open code freeze?

Reply:
agreed, will add this to the next PR